src/Combinatorics/Graphs/functions.jl

```diff
@@ -1687,3 +1687,27 @@ true
 function is_bipartite(g::Graph{Undirected})
   return Polymake.graph.Graph{Undirected}(ADJACENCY=pm_object(g)).BIPARTITE::Bool
 end
+
+@doc raw"""
+    maximal_cliques(g::Graph{Undirected})
+
+Returns the maximal cliques of a graph `g` as a `Vector{Set{Int}}`.
+
+# Examples
+```jldoctest
+julia> g = graph_from_edges([[1, 2], [2, 3], [1, 3], [2, 4], [3, 4]])
+Undirected graph with 4 nodes and the following edges:
+(2, 1)(3, 1)(3, 2)(4, 2)(4, 3)
+
+julia> maximal_cliques(g)
+2-element Vector{Set{Int64}}:
```

Comment:
shouldn't this be a `Set{Set{Int}}`? At least this is what you coerce it into in line 1710, and IMO it makes more sense here